src/player.h

```diff
@@ -1166,6 +1166,10 @@ class Player final : public Creature, public Cylinder
 		void forgetInstantSpell(const std::string& spellName);
 		bool hasLearnedInstantSpell(const std::string& spellName) const;
 
+		uint64_t getLastPong() const {
+			return lastPong;
+		}
```

Comment:
```
player.h
int64_t lastPong;
```

Reply:
lastPing and lastPong should be stored in u64, so this should be changed, but thats another pr

Reply:
> ~u64~

`std::chrono::time_point`